joss_paper_0.18/paper.bib

```diff
@@ -0,0 +1,38 @@
+@SciPy{,
+  author =    {Eric Jones and Travis Oliphant and Pearu Peterson and others},
+  title =     {{SciPy}: Open source scientific tools for {Python}},
+  year =      {2001--},
+  url = "http://www.scipy.org/"
+}
+
+@Article{NumPy,
+  author =       {Stéfan {van der Walt} and S. Chris Colbert and
+                  Gaël Varoquaux},
+  title =        {The NumPy Array: A Structure for Efficient Numerical
+                  Computation},
```

Comment:
Actually, I'm much more convinced of changing the NumPy one rather than the SciPy one. A citation for NumPy that doesn't include Travis and Chuck makes little sense.